packages/daemon/src/daemon-node-powers.js

```diff
@@ -367,6 +367,7 @@ export const makeDaemonicPersistencePowers = (
           return filePowers.readFileText(storagePath);
         };
         const json = async () => {
+          await null;
           return JSON.parse(await text());
```

Comment:
This is an odd case because the text I’m suggesting has equivalent behavior but avoids the semblance of naughtiness with nothing more than a tortuously named intermediate variable.

Reply:
Done